src/config/limits.ts

```
@@ -1,25 +1,177 @@
+    Date.now() - cacheTimestamp < CACHE_TTL_MS
+  ) {
+    return limitsCache[audience];
+  }
+
+  // Refresh cache
+  limitsCache = await fetchLimitsFromDb();
+  cacheTimestamp = Date.now();
+  return limitsCache[audience];
+}
+
+export async function updateLimit(audience: Audience, partial: Partial<LimitConfig>): Promise<LimitConfig> {
```

ESLint

[error] Replace `audience:·Audience,·partial:·Partial<LimitConfig>` with `⏎··audience:·Audience,⏎··partial:·Partial<LimitConfig>,⏎`

(prettier/prettier)